docs/website/binding_types_reference_guide.md

```diff
@@ -1380,7 +1380,18 @@ static library, use "__Internal" as the `libraryName` parameter.
 
 The generated properties are always static.
 
-Properties flagged with the Field attribute can be of type `NSString`, `NSArray`, `nint`, `double`, `nfloat` or `System.IntPtr`.
+Properties flagged with the Field attribute can be of the following types:
+
+* `NSString`
+* `NSArray`
+* `nint` / `int` / `long`
+* `nuint` / `uint` / `ulong`
+* `nfloat` / `float`
+* `double`
+* `CGSize`
+* `System.IntPtr`
+* Enums
+* [Enums backed by a NSString constant](#enum-attributes), setter is not supported.
```

Comment:
is -> are (plural)

It might be better to remove the last from the list (and make it a separate paragraph) since it's a special case. E.g.

`For [enums backed by NSString constants](#enum-attributes) the setters are not supported.`

Reply:
It would also be a nice addition to mention that that can be bound manually (if needed). :)